opennms-config-model/src/main/java/org/opennms/netmgt/xml/eventconf/EventMatchers.java

```diff
@@ -191,6 +191,19 @@ public String toString() {
 	}
 	
 	public static Field field(String name) {
+
+        if (name.startsWith("parm[") && name.endsWith("]")) {
+            String parmName = name.substring(5, name.length() - 1);
```

Comment:
```suggestion
            String parmName = name.substring("parm[".length());
```

Reply:
Oops. I was mostly trying to get rid of the magic number 5 in the call to substring. But it doesn't really matter. Can ignore that one.